main/src/modules/Jvm.scala

```diff
@@ -335,16 +311,14 @@ object Jvm {
     PathRef(output)
   }
 
-  private def writeEntry(p: java.nio.file.Path, is: InputStream, append: Boolean): Unit = {
```

Comment:
We don't need to append now since we are doing the `InputStreams` concat and we are not merging with the upstream jar.